### PR TITLE
Effect of using HTTPS to twart MITM changed, no "we"

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: securedrop/source_templates/tor2web-warning.html [hunk 1]
```diff
@@ -2,6 +2,6 @@
 {% block body %}
 <h1>Why is there a warning about Tor2Web?</h1>
 <p><a href='https://tor2web.org'>Tor2Web</a> is a proxy service that lets you browse Tor Hidden Services (.onion sites) without installing Tor. It was designed to facilitate anonymous publishing.</p>
-<p>Tor2Web only protects publishers, not readers. If you upload documents to us using Tor2Web, you are <strong>not anonymous</strong> and could be identified by your ISP or the Tor2Web proxy operators. Additionally, since Tor2Web sites typically use HTTPS, it is possible that your connection could be MITM'ed by a capable adversary.</p>
-<p>If you want to submit information, we <strong>strongly advise you</strong> to install <a href='https://www.torproject.org/download.html.en'>Tor</a> and use it to access our site safely and anonymously.</p>
+<p>Tor2Web only protects publishers, not readers. If you upload documents to us using Tor2Web, you are <strong>not anonymous</strong> and could be identified by your ISP or the Tor2Web proxy operators. Additionally, since Tor2Web sites typically do not use HTTPS, it is possible that your connection could be MITM'ed by a capable adversary.</p>
```
Comment (dachary) [inline securedrop/source_templates/tor2web-warning.html]: **Tor2Web sites typically do not use HTTPS** 

Good catch !

**Comment by dachary** [comment]
https://circleci.com/gh/freedomofpress/securedrop/1752?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

<pre>
fatal: [app-staging -> localhost]: FAILED! => {"changed": false, "elapsed": 601, "failed": true, "msg": "Timeout when waiting for search string OpenSSH in ec2-13-56-20-40.us-west-1.compute.amazonaws.com:22"}
</pre>
amending and re-pushing will retry another run.

**Comment by conorsch** [comment]
Changes LGTM. CI still failing, so kicked it off again.

**Comment by conorsch** [comment]
This PR should not target master, it should target the develop branch. See here: https://docs.securedrop.org/en/latest/development/contributor_guidelines.html

@comradekingu Can you update this PR to target develop? Will take another look at merging then.

**Comment by dachary** [comment]
continued at https://github.com/freedomofpress/securedrop/pull/2279

**Pull request closed without merging**